Server/BackEndMonitoringServer/BackEndMonitoringServer.vcxproj

```diff
@@ -171,6 +171,7 @@
     <ClInclude Include="..\..\3rdParty\include\pugiconfig.hpp" />
```

Comment:
@AntonVoytov Sorry for such big changes in code, this was the only way to return structures of data(as i think).
There are 37 warnings in pugixml library, i hope that is okey. I've added a lot of hardcode in order to return structures of data. But, there still used the same patterns. Hope that is ok. If not, then I hope You can tell me how best to do it, as I don't have another ideas, how to read certain data, without hardcode.